cl/recap/utils.py

```diff
@@ -22,17 +24,14 @@ def get_court_id_from_fetch_queue(fq: PacerFetchQueue | dict[str, Any]) -> str:
     PacerFetchQueue.
     :return: The court ID as a string.
     """
-    # Check if the input is a PacerFetchQueue object or a dictionary.
-    is_fetch_queue = isinstance(fq, PacerFetchQueue)
```

Comment:
Could you explain why these lines are being removed? I've reviewed the changes and it appears the only difference is the replacement of this variable with isinstance checks

Reply:
Sure! Since I added cl.recap.utils.py to mypy linting, it started complaining about `attrs` being a dictionary. It seems that mypy was not able to infer the type based on the variable `is_fetch_queue`, but it did when using the condition `isinstance(fq, PacerFetchQueue)` directly.